wiki/en/Software-Manual.md

```diff
@@ -240,9 +240,9 @@ rate is not higher than your available internet upload speed (check this with a
 
 <figure><img src="{% include img/en-screenshots/settings-advanced.inc %}" style="border: 5px solid grey;" loading="lazy" alt="Image of advanced settings window"></figure>
 
-### Custom directory server address
+### Custom Directory server address
```

Comment:
![image](https://user-images.githubusercontent.com/1549463/182020925-5acb285b-5220-4e98-b1f0-ff0c9ca97128.png)
"Custom Directory address", in fact.